src/mofaflex/_core/mofaflex.py

```diff
@@ -289,6 +301,26 @@ def _mofaflexdataset(self, data: MuData | dict[str, dict[str, AnnData]]) -> Mofa
         self._make_preprocessor(data)
         return data
 
+    @property
+    def n_guided_factors(self) -> int:
+        """Number of guided factors."""
+        return self._n_guiding_vars
+
+    @property
+    def guiding_vars_n_categories(self) -> dict[str, int]:
+        """Number of categories for each guiding variable (1 for non-categorical data)."""
+        return self._guiding_vars_n_categories
+
+    @property
+    def guiding_vars_factors(self) -> dict[str, dict[str, str]]:
```

Comment:
I feel like this should either be removed or a similar property offered for informed factors.